ocaml/stdlib/obj.mli

```diff
@@ -172,3 +172,35 @@ module Ephemeron: sig
   (** Maximum length of an ephemeron, ie the maximum number of keys an
       ephemeron could contain *)
 end
+
+module Uniform_or_mixed : sig
+  (** Blocks with a nominally scannable tag can still have a suffix of
+      unscanned objects; such a block is "mixed". This contrasts with
+      "uniform" blocks which are either all-scanned or all-unscanned.
+  *)
```

Comment:
Maybe it's worth noting somewhere in here that these functions can return different results on nativecode and bytecode, which might be a surprising source of test failures to someone who hasn't thought about the bytecode representation?

Reply:
Added a comment about this, thanks.